javascript-methods/javascript-methods-quiz-notes.md

```diff
@@ -0,0 +1,59 @@
+# javascript-methods-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Why do we log things to the console?
+console.log()
+- What is a method?
+function which is a property of an object, make object do something
+- How is a method different from any other function?
+methods are built in
+- How do you remove the last element from an array?
+array.pop()
+- How do you round a number down to the nearest integer?
+math.floor()
+- How do you generate a random number?
+math.floor(math.random(array.length-1))
```

Comment:
`Math.floor` only rounds, it doesn't have anything to do with generating a random number.